### ISSUE TITLE
Console.log has no limit on objects

### ISSUE THREAD

**Issue by MusicMakerOwO** [oven-sh/bun#5238]
### What version of Bun is running?

1.0.1+31aec4ebe325982fc0ef27498984b0ad9969162b

### What platform is your computer?

Linux (Ubuntu)

### What steps can reproduce the bug?

In my specific case, using discord.js
`
client.on('mesageCreate', function (message) {
    console.log(message);
});
`

In a general case:
1) Create any deeply nested object
2) Print it to console

Bonus if you use a class reference or even multiple, every single one gets printed :P

### What is the expected behavior?

Standard NodeJS uses util.inspect, depth of 3
Fine with it being bumped up 1 but not everything lol

### What do you see instead?

Literally everything lmao
Video: https://drive.google.com/file/d/1UtbwZp9F_cbtTD4NASU_CDsISpxJS9gx/view?usp=sharing

**Comment by JibranKalia** [comment]
So currently the default depth set in Bun is 8 [Source](https://github.com/oven-sh/bun/blob/972a6f29cc0d232982ad9d2cae493c8a5cc7a74a/src/bun.js/bindings/exports.zig#L1045)

One workaround could be:

```
console.log(require('node:util').inspect(deeplyNestedObject, { depth: 3 }));
```

**Comment by JibranKalia** [comment]
This should be fixed by https://github.com/oven-sh/bun/pull/5839

**Comment by Electroid** [comment]
This was fixed by #5839
